### DISCUSSION TITLE
Windows + Docker Reload not Working

### DISCUSSION THREAD

**Discussion by kddove85** [Kludex/uvicorn#1980] [Potential Issue]
I have developed an app using Django, which incorporates Vue and is deployed with Docker. To launch the app, the docker-compose file uses the command below:
```
command: "uvicorn django-app.asgi:application --reload --reload-include *.vue --host 0.0.0.0 --port ${APP_PORT:-8000}"
```
The app's home page has a vue file, and whenever I make any edits to it, the following messages appear:
```
django-app-api-1 | WARNING: WatchFiles detected changes in 'assets/src/views/HomeView.vue'. Reloading...
django-app-api-1 | INFO: Shutting down
django-app-api-1 | INFO: Finished server process [11]
django-app-api-1 | INFO: Started server process [27]
django-app-api-1 | INFO: Waiting for application startup.
django-app-api-1 | INFO: ASGI 'lifespan' protocol appears unsupported.
django-app-api-1 | INFO: Application startup complete.
```
Although it seems that the application has restarted, when I refresh the page, I do not see any changes. The page looks exactly the same as before, and no modifications have been detected.

I attempted to include watchfiles in my requirements file with the following code:
```
uvicorn[standard]~=0.14
watchfiles==0.19.0
```
When it seems like the solution didn't work, I attempted to add the environment variable for force polling by including:
```
WATCHFILES_FORCE_POLLING=True
```
in the .env file. And I've tried disabling the cache in my browser. I'm at my wit's end with this. I don't know what else to try. Any ideas?

**Comment by parthvnp** [comment]
@kddove85  I might be able to help this. Does the project's configuration match any of [these](https://vue-loader.vuejs.org/guide/hot-reload.html#disabling-hot-reload) conditions that might disable the changes? Also, does hot reloading work through `vue-cli` itself?

**Reply by kddove85** [reply]
No. None of the configurations are what I have. I did have the NODE_ENV == production, but I changed it to development and it still didn't do anything. I'm pretty confident that the issue is with uvicorn and watchfiles/watchgod.

**Reply by parthvnp** [reply]
Gotcha. Can you share the Dockerfile and a minimal reproducible example if possible ?
